test/util/http/common.js

```diff
@@ -99,6 +99,11 @@ export function testModalToggles({
   // shown
   respond = (series) => series
 }) {
+  if (typeof modal === 'string')
+    throw new Error('modal must be a component, not a string');
```

Comment:
I needed to change `modal: '#web-form-renderer-submission-modal'` to `modal: Modal` above. `.getComponent()` doesn't seem to play well with a string when working with modals.